lib/event_summarizer/vendor_result_evaluators/aamva.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module EventSummarizer
+  module VendorResultEvaluators
+    module Aamva
+      ID_TYPES = {
+        'state_id_card' => 'non-driving ID card',
+        'drivers_license' => 'drivers\' license',
```

Comment:
might be easier as double quotes?
```suggestion
        'drivers_license' => "drivers' license",
```